File: JavaScript/JavaScript-Fundamentals/Operators and Expressions/Operators and Expressions/ThirdDigit.js
function thirdDigit(num) {
    var third = Math.floor(num / 100) % 10;
    return third === 7;
}
console.log('***********************************');
console.log('Problem 4: Third Digit = 7');
console.log('***********************************');

console.log("5: " + thirdDigit(5));
console.log("701: " + thirdDigit(701));
console.log("1732: " + thirdDigit(1732));
console.log("9703: " + thirdDigit(9703));
console.log("877: " + thirdDigit(877));
console.log("777877: " + thirdDigit(777877));
console.log("9999799: " + thirdDigit(9999799));

File: CSharp1/OperatorsAndExpressions/ThirdDigit/ThirdDigit.cs
/* Problem 5: Write an expression that checks for given integer if its third digit from right-to-left is 7. */

using System;

class ThirdDigit
{
    static void Main()
    {
        Console.WriteLine("Enter a number: ");
        int num = int.Parse(Console.ReadLine());

        if ((num / 100) % 10 == 7) 
        {
            Console.WriteLine("The 3rd digit of the number you entered is 7");
        }
        else
        {
            int digit = ((num / 100) % 10);
            Console.WriteLine("The 3rd digit of the number you entered ISN'T 7. It's: {0}", digit);
        }

    }
}